ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.12.yaml

```diff
@@ -141,12 +141,12 @@ tests:
         group\|The OpenStack platform creates Control plane nodes in a server group
       TEST_SUITE: experimental/reliability/minimal
     workflow: openshift-e2e-openstack-upi
-- as: e2e-openstack-appcreds
+- as: e2e-openstack-password
   minimum_interval: 24h
   steps:
     cluster_profile: openstack-vexxhost
     env:
-      APPLICATION_CREDENTIALS: "true"
+      PASSWORD_AUTHENTICATION: "true"
       TEST_SKIPS: 'In-tree Volumes \[Driver: cinder\]'
```

Comment:
When switching our jobs to use app creds by default, we need to skip the in-tree cinder tests (as we were doing here) as the cinder executable does not work with app creds. Possibly something we'll want [at a higher level in origin](https://github.com/openshift/origin/blob/master/test/extended/util/annotate/rules.go).